src/utils/restApiWrapper.ts

```diff
@@ -0,0 +1,58 @@
+export async function postData(
+    url = '',
+    headers = {},
+    data = {},
+): Promise<any> {
+    const response = await fetch(url, {
+        method: 'POST',
+        mode: 'no-cors',
+        headers: {
+            ...headers,
+            'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(data),
+    })
+    return response
+}
+
+export async function getData(url = '', headers = {}): Promise<any> {
+    const response = await fetch(url, {
+        method: 'GET',
+        mode: 'no-cors',
+        headers: {
+            ...headers,
+            'Content-Type': 'application/json',
+        },
+    })
+    return response
+}
+
+export async function putData(url = '', headers = {}, data = {}): Promise<any> {
```

Comment:
could you add some function documentation to all of these :)

Reply:
Done, I've done it the typescript way by making the function arguments more strictly type checked 😄